vignettes/ReactomeContentService4R.Rmd

```diff
@@ -231,19 +231,62 @@ getOrthology("R-SSC-69541", species = "human")
 
 The Reactome [diagram exporter](https://reactome.org/dev/content-service/diagram-exporter) has been mentioned a little bit in 'Instances in Reactions' section, here we can reconstruct a few more examples.
 
-Export animaged gifs of Enhanced High Level Diagrams ([**EHLDs**](https://reactome.org/icon-info/ehld-specs-guideline)) of Pathway "Cell Cycle, Mitotic" (stId R-HSA-69278) with overlaid analysis results, and highlight "S Phase" (stId R-HSA-69242):
+What's more, Reactome offers a pathway [Analysis Service](https://reactome.org/dev/analysis) that supports enrichment and expression analysis. The diagram exporter allows you to overlay the results of the analysis on top of the exported diagrams. To do so, use the token argument to specify the unique token associated with the performed analysis.
 
-```{r gif}
-gif <- exportImage(id = "R-HSA-69278", output = "diagram", format = "gif",
-                   sel = "R-HSA-69242", token = "MjAyMDA2MTcyMDM5NDBfMzU2")
-print(gif)
+The [ReactomeGSA](https://github.com/reactome/ReactomeGSA) package is an R client to the web-based Reactome Analysis Service, so we could perform analysis in R then access the token.
+
+```{r ReactomeGSA, eval=FALSE}
+library(ReactomeGSA)
+library(ReactomeGSA.data)
+data("griss_melanoma_proteomics")
+
+# Create an analysis request and set parameters
+my_request <- ReactomeAnalysisRequest(method = "Camera")
+my_request <- set_parameters(request = my_request, max_missing_values = 0.5)
+my_request <- add_dataset(request = my_request, 
```

Comment:
I have trouble running this command, and I think its related to the installlation of ReactomeGSA that I have. I followed the bioconductor instructions but I get this error: 

```
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘add_dataset’ for signature ‘"ReactomeAnalysisRequest", "character"’
```

Reply:
Have you installed `ReactomeGSA.data`?

Reply:
Yes, using `BiocManager::install("ReactomeGSA.data")`. Looking over the install logs, it looks like there was an issue installing/compiling 'mgcv'

Reply:
Oh `ReactomeGSA.data` may fail to install